tools/docker/Dockerfile

```diff
@@ -1,6 +1,23 @@
 FROM alpine
 ARG TARGETARCH
-RUN apk add --no-cache --verbose ca-certificates && adduser -D kopia && addgroup kopia kopia
+
+# install CA certificates and create user/group 'kopia' with IDs 51515:51515
+RUN apk add --no-cache --verbose ca-certificates && \
```

Comment:
What are the certs for?

Reply:
mostly connecting to S3, Azure and other services on the internet.